packages/flutter_tools/lib/src/http_host_validator.dart

```diff
@@ -56,69 +55,57 @@ class HttpHostValidator extends DoctorValidator {
 
   /// Make a head request to the HTTP host. HTTP Host is available if no exception happened
   Future<_HostValidationResult> _checkHostAvailability(String host) async {
+    final int timeout = int.tryParse(_platform.environment[kDoctorHostTimeout] ?? '10') ?? 10;
     try {
-      final int timeout =
-      int.parse(_platform.environment[kDoctorHostTimeout] ?? '10');
       final HttpClientRequest req = await _httpClient.headUrl(Uri.parse(host));
       await req.close().timeout(Duration(seconds: timeout));
-      // HTTP Host is available if no exception happened
+      // HTTP host is available if no exception happened
       return _HostValidationResult.success(host);
     } on TimeoutException {
-      return _HostValidationResult.fail(
-          host, 'Failed to connect to $host in seconds');
+      return _HostValidationResult.fail(host, 'Failed to connect to $host in $timeout seconds');
     } on SocketException catch (e) {
-      return _HostValidationResult.fail(
-          host, 'An error occurred while checking the HTTP host: ${e.message}');
+      return _HostValidationResult.fail(host, 'An error occurred while checking the HTTP host: ${e.message}');
     } on HttpException catch (e) {
-      return _HostValidationResult.fail(host,
-          'An error occurred while checking the HTTP host: ${e.toString()}');
+      return _HostValidationResult.fail(host, 'An error occurred while checking the HTTP host: ${e.toString()}');
     } on OSError catch (e) {
-      return _HostValidationResult.fail(
-          host, 'An error occurred while checking the HTTP host: ${e.message}');
+      return _HostValidationResult.fail(host, 'An error occurred while checking the HTTP host: ${e.message}');
     }
   }
 
   @override
   Future<ValidationResult> validate() async {
     final List<ValidationMessage> messages = <ValidationMessage>[];
-    final Iterable<Future<_HostValidationResult>> availabilityResultFutures =
-    _requiredHosts.map(_checkHostAvailability);
 
-    final List<_HostValidationResult> availabilityResults =
-    (await Future.wait(availabilityResultFutures)).toList();
+    final List<_HostValidationResult> availabilityResults = await Future.wait(_requiredHosts.map(_checkHostAvailability));
```

Comment:
Not sure why the `.toList` was needed here

Reply:
Why did you condense lines 89-92 into one? I feel like it is more difficult to read as a one-liner.

Reply:
So should I reintroduce `availabilityResultFutures`?

Reply:
If it was just a style refactor, I would prefer having `availabilityResultFutures`

Reply:
I agree that `.toList()` was redundant